kiwi/bootloader/config/bootloader_spec_base.py

```diff
@@ -89,6 +89,19 @@ def setup_sysconfig_bootloader(self) -> None:
         """
         pass
 
+    def write_meta_data(
```

Comment:
What calls this function?

Reply:
The builder. https://github.com/OSInside/kiwi/blob/main/kiwi/builder/disk.py#L1462